09-mapping.Rmd

```diff
@@ -291,21 +313,24 @@ The `equal` style divides the range of input values into subranges of equal rang
 This style is often used on data with regular distribution of values and common values range, such as percentages.
 On the other hand, it could produce categories with a large number of elements (points, areas) and those without any observation.
 To have the same number of elements in each category, one can use the `quantile` style. 
-It is useful for ordinal data <!--examples--> and never produce empty classes. 
+It is useful for ordinal data <!--examples--> and never produces empty classes. 
 This style, however, could have very different values in the same category.
 Natural breaks method, also called `jenks`, tries to identify groups of similar values in the data.
 Breaks between classes are created to maximize the differences between categories. 
-This method provides classes that reflect the values distribution, but in the same creates classes that could be meaningful only for this dataset.
+<!-- what do you mean by could be meaningful only for this? Also "Importantly" sentence sounds strange -->
+This method provides classes that reflect the values distribution, but at the same time creates classes that could be meaningful only for this dataset.
 Importantly, categorical legends described above could be not the most suitable when presenting a large number of objects (e.g. points or polygons) or continuous fields (e.g. continuous rasters).
-In this situations, `cont` or `order` can be used. 
+In these situations, `cont` or `order` can be used. 
 The former is appropriate for variables with evenly distributed values, while the latter works well when the distribution of values is skewed (for example, only a few raster cells have very large values).
 Finally, the `cat` style should be used to represent categorical values.
-It assures that each unique value will be presented by its own category, and therefore color.
+It assures that each category receives a unique color.
 <!-- references for more info -->
 
 ### Layouts
 
-Layout refers not to the map itself but to its wider surroundings.
+The map layout refers to the combination of all map elements into a cohesive map.
+Map elements include among others the objects to be mapped, the title, the scale bar, margins and aspect ratios.
+<!-- what is the relation of the next two sentences to the general map layout. It sounds more like these should be moved to the palette and break sections. -->
 Color settings relate to the palette and break-points used affect how the map looks.
```

Comment:
What is the relation of these two sentences to the general map layout? It sounds more like these should be moved to the palette and break sections.